packages/braintree-integration/src/braintree-paypal/braintree-paypal-customer-strategy.ts

```diff
@@ -1,78 +1,78 @@
-import mapToLegacyShippingAddress from '../../../checkout-buttons/strategies/braintree/map-to-legacy-shipping-address';
 import {
+    CustomerCredentials,
+    CustomerInitializeOptions,
+    CustomerStrategy,
+    ExecutePaymentMethodCheckoutOptions,
     InvalidArgumentError,
     MissingDataError,
     MissingDataErrorType,
+    PaymentIntegrationService,
+    PaymentMethod,
+    RequestOptions,
     StandardError,
-} from '../../../common/error/errors';
-import { PaymentMethodActionCreator } from '../../../payment';
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
 import {
     BraintreeError,
+    BraintreeHostWindow,
+    BraintreeInitializationData,
     BraintreePaypalCheckout,
     BraintreePaypalSdkCreatorConfig,
-    BraintreeSDKCreator,
     BraintreeTokenizePayload,
-    mapToBraintreeShippingAddressOverride,
-} from '../../../payment/strategies/braintree';
-import { PaypalAuthorizeData, PaypalHostWindow } from '../../../payment/strategies/paypal';
-import CustomerActionCreator from '../../customer-action-creator';
-import CustomerCredentials from '../../customer-credentials';
-import {
-    CustomerInitializeOptions,
-    CustomerRequestOptions,
-    ExecutePaymentMethodCheckoutOptions,
-} from '../../customer-request-options';
-import CustomerStrategy from '../customer-strategy';
+} from '../braintree';
+import BraintreeIntegrationService from '../braintree-integration-service';
+import isBraintreeError from '../is-braintree-error';
+import { PaypalAuthorizeData } from '../paypal';
 
-import BraintreePaypalCustomerInitializeOptions from './braintree-paypal-customer-options';
+import BraintreePaypalCustomerInitializeOptions, {
+    WithBraintreePaypalCustomerInitializeOptions,
+} from './braintree-paypal-customer-options';
 
 export default class BraintreePaypalCustomerStrategy implements CustomerStrategy {
     constructor(
-        private _store: CheckoutStore,
-        private _checkoutActionCreator: CheckoutActionCreator,
-        private _customerActionCreator: CustomerActionCreator,
-        private _paymentMethodActionCreator: PaymentMethodActionCreator,
-        private _braintreeSDKCreator: BraintreeSDKCreator,
-        private _formPoster: FormPoster,
-        private _window: PaypalHostWindow,
+        private paymentIntegrationService: PaymentIntegrationService,
+        private formPoster: FormPoster,
+        private braintreeIntegrationService: BraintreeIntegrationService,
+        private braintreeHostWindow: BraintreeHostWindow,
     ) {}
 
-    async initialize(options: CustomerInitializeOptions): Promise<InternalCheckoutSelectors> {
+    async initialize(
+        options: CustomerInitializeOptions & WithBraintreePaypalCustomerInitializeOptions,
+    ): Promise<void> {
         const { braintreepaypal, methodId } = options;
-        const { container, onError, buttonHeight = 40 } = braintreepaypal || {};
+        const { container, buttonHeight, onError } = braintreepaypal || {};
```

Comment:
Is default height is needed?

Reply:
oh see, it below

Reply:
This height was added by checkout team, I think it is default option for all buttons on customer step
But in future styles for braintree buttons will get from initialisation data